src/App.js

```diff
@@ -55,14 +124,14 @@ const App = () => {
     <div className="App">
       <header className="App-header">
         <h1>React Tic Tac Toe</h1>
-        <h2>The winner is ... -- Fill in for wave 3 </h2>
+        <h2>The winner is ... {winner} </h2>
```

Comment:
The winner detection code works, but this section is causing the tests to fail.  The tests are looking for a very specific string ("Winner is x" or "Winner is o") and capitalization and punctuation matter.  A very minor change here will result in the tests passing:
```suggestion
        <h2>Winner is {winner} </h2>
```